gui/themes/common/lowres_layout.stx

```diff
@@ -2150,6 +2150,15 @@
 					width = '480'
 					height = '250'
 			/>
+			<layout type = 'horizontal' padding = '-80, 8, 8, 8'>
```

Comment:
-80 is like 1/4 of a 320 pixels wide screen. Doesn't feel right. Could you please post a screenshot?